indico/modules/events/registration/controllers/management/regforms.py

```diff
@@ -36,7 +36,8 @@
 from indico.modules.events.registration.models.items import PersonalDataType
 from indico.modules.events.registration.models.registrations import Registration
 from indico.modules.events.registration.stats import OverviewStats, AccommodationStats
-from indico.modules.events.registration.util import get_event_section_data, create_personal_data_fields
+from indico.modules.events.registration.util import get_event_section_data, create_personal_data_fields, \
```

Comment:
We usually wrap the imports in parenthesis when they use more than one line, exactly as you have in the line below.